dashboard/app/models/concerns/user/level_progressable.rb

```diff
@@ -0,0 +1,156 @@
+module User::LevelProgressable
```

Comment:
I'd love to see a brief comment here describing the functionality provided by this concern; could even be as brief as "methods related to level progression"

Reply:
Adding that!